StitchApp/WorkerPoolStaging/functions/getUsernameMapping/source.js

```diff
@@ -0,0 +1,5 @@
+exports = function(arg) {
+  return {
+    // username => email mapping
```

Comment:
I'm assuming this is blank to protect identities....

Reply:
yes :)